ckanext/switzerland/logic.py

```diff
@@ -46,8 +46,15 @@ def ogdch_dataset_count(context, data_dict):
         {'rows': 0, 'fq': '+dataset_type:dataset'}
     )
 
+    # get the total number of showcases from package_search
```

Comment:
Is it also possible to call the helper-function here instead of requesting the count all over again?
Even though I see that the `search_result` also does just another request which leads with bad example :)

But if possible I would prefer just calling the helper function here, so if anything changes we don't have to do it in multiple places.